__snapshots__/generate-devsite.mjs.js

```diff
@@ -1723,9 +1723,9 @@ items:
         - id: opts
           description: ''
           type:
-            - 'google-gax!ClientOptions:interface'
```

Comment:
Some lines are surrounded with quotes, other are not (i.e. `uid` entries vs `type`, `summary`, etc). Is there a reason for some of them to be with quotes and others without? It could just be that I'm not familiar with Node's way of dealing with this.

Reply:
I think this is caused by a couple of things:

+ In many cases, YAML doesn't care whether or not you enclose a string in quotes.
+ We're now parsing and stringifying each YAML file with the [`js-yaml` package](https://github.com/nodeca/js-yaml), which might have different quoting rules than whatever library `api-documenter` is using.

If I'm correct, then these differences shouldn't affect the generated docs. But I might be wrong. If someone who's more familiar with the DocFX converter could use this PR to generate some YAML files, then run them through the rest of the doc-generation pipeline, that would be helpful!

Reply:
I ran this through docfx, and I see, with our template at least, the `name`, `fullName`, and `summary` keys are not affected by being wrapped by quotes or not.